src/components/modules/Feedback/FormRating/index.tsx

```diff
@@ -0,0 +1,165 @@
+      name: 'Trần Quang Hiệp',
+      age: 28,
+      position: 'Tiền đạo, Hậu vệ',
+      competitiveness: 'Vui vẻ',
+      phoneNumber: '987654321',
+    },
+    {
+      id: 3,
+      avatar: 'images/layout/avatar.jpeg',
+      name: 'Trần Quang Hiệp',
+      age: 28,
+      position: 'Tiền đạo, Hậu vệ',
+      competitiveness: 'Vui vẻ',
+      phoneNumber: '987654321',
+    },
+    {
+      id: 4,
+      avatar: 'images/layout/avatar.jpeg',
+      name: 'Trần Quang Hiệp',
+      age: 28,
+      position: 'Tiền đạo, Hậu vệ',
+      competitiveness: 'Vui vẻ',
+      phoneNumber: '987654321',
+    },
+    {
+      id: 5,
+      avatar: 'images/layout/avatar.jpeg',
+      name: 'Trần Quang Hiệp',
+      age: 28,
+      position: 'Tiền đạo, Hậu vệ',
+      competitiveness: 'Vui vẻ',
+      phoneNumber: '987654321',
+    },
+    {
+      id: 6,
+      avatar: 'images/layout/avatar.jpeg',
+      name: 'Trần Quang Hiệp',
+      age: 28,
+      position: 'Tiền đạo, Hậu vệ',
+      competitiveness: 'Vui vẻ',
+      phoneNumber: '987654321',
+    },
+    {
+      id: 7,
+      avatar: 'images/layout/avatar.jpeg',
+      name: 'Trần Quang Hiệp',
+      age: 28,
+      position: 'Tiền đạo, Hậu vệ',
+      competitiveness: 'Vui vẻ',
+      phoneNumber: '987654321',
+    },
+    {
+      id: 8,
+      avatar: 'images/layout/avatar.jpeg',
+      name: 'Trần Quang Hiệp',
+      age: 28,
+      position: 'Tiền đạo, Hậu vệ',
+      competitiveness: 'Vui vẻ',
+      phoneNumber: '987654321',
+    },
+    {
+      id: 9,
+      avatar: 'images/layout/avatar.jpeg',
+      name: 'Trần Quang Hiệp',
+      age: 28,
+      position: 'Tiền đạo, Hậu vệ',
+      competitiveness: 'Vui vẻ',
+      phoneNumber: '987654321',
+    },
+    {
+      id: 10,
+      avatar: 'images/layout/avatar.jpeg',
+      name: 'Trần Quang Hiệp',
+      age: 28,
+      position: 'Tiền đạo, Hậu vệ',
+      competitiveness: 'Vui vẻ',
+      phoneNumber: '987654321',
+    },
+    {
+      id: 11,
+      avatar: 'images/layout/avatar.jpeg',
+      name: 'Trần Quang Hiệp',
+      age: 28,
+      position: 'Tiền đạo, Hậu vệ',
+      competitiveness: 'Vui vẻ',
+      phoneNumber: '987654321',
+    },
+  ];
+
+  const toggleSelection = (playerId: number) => {
+    if (selectedPlayers.includes(playerId)) {
+      setSelectedPlayers(selectedPlayers.filter((id) => id !== playerId));
+    } else {
+      setSelectedPlayers([...selectedPlayers, playerId]);
+    }
+  };
+
+  const handleSubmit = () => {};
+
+  return (
+    <>
+      <S.RatingTitle style={{ margin: '18px 0px' }}>
+        Chọn để tố cáo sủi kèo
+      </S.RatingTitle>
+      <S.PlayerWrapper>
+        {players.map((player) => (
+          <S.CustomButton
+            key={player.id}
+            onClick={() => toggleSelection(player.id)}
+            selected={selectedPlayers.includes(player.id)}
+          >
+            <S.PlayerFlex>
+              <S.PlayerItemOrder>{player.id}</S.PlayerItemOrder>
```

Comment:
những data này check null em nhé. {player.id} ->  {player?.id}. Check lại toàn bộ source